src/product.html

```diff
@@ -541,9 +541,9 @@ <h3 id="location-store-front">{{item.results[0].shop_location}}</h3>
         <!-- endbower -->
         <script src="js/plugins.js"></script>
         <script src="js/main.js"></script>
-        <script src="js/store-front.js"></script>
-        <script src="js/image-slider.js"></script>
-        <script src="js/listing-header.js"></script>
+        <script src="js/product-data.js"></script>
+        <!-- <script src="js/image-slider.js"></script>
+        <script src="js/listing-header.js"></script> -->
```

Comment:
Just take the commented lines out, and instead of naming the file `product-data.js`, how about naming it to match the module name?